fontforge/mm.c

```diff
@@ -1130,14 +1130,14 @@ return( false );
 			if ( complain ) {
 			    FVChangeGID( sf->fv,i);
 			    ff_post_error(_("Bad Multiple Master Font"),_("The %1$s hints in glyph \"%2$.30s\" in font %3$.30s do not match those in %4$.30s (different number or different overlap criteria)"),
-				    "horizontal", sf->glyphs[i]->name,sf->fontname, mm->instances[j]->fontname);
+				    _("horizontal"), sf->glyphs[i]->name,sf->fontname, mm->instances[j]->fontname);
```

Comment:
Splitting localizable strings is a bad practice, as they cannot be reliably translated due to syntax differences. This string should be as follows:
`_("The horizontal hints in glyph \"%1$.30s\" in font %2$.30s do not match those in %3$.30s (different number or different overlap criteria)"), sf->glyphs[i]->name,sf->fontname, mm->instances[j]->fontname`

The other string below should be changed accordingly.

Reply:
Oh, yes! Especially in this case substitution makes no sense.